exporters/holstebro_helpers.py

```diff
@@ -0,0 +1,811 @@
+                  'mobile',
+                  'loen_nummer',
+                  'forvaltning',  # direktørområde
+                  'omraade',  # chefområde
+                  'afdeling',  # nærmeste leders afdeling
+                  'gruppe',  # engagmentsafdeling
+                  'org_sti',
+                  'stilling',
+                  'engagements_type']
+
+    rows = []
+
+    for node in PreOrderIter(nodes['root']):
+        ou = mh.read_ou(node.name)
+        # Do not add "Afdelings-niveau"
+        if ou['org_unit_level']['name'] != 'Afdelings-niveau':
+            employees = mh.read_organisation_people(node.name, 'engagement', False)
+            manager = find_org_manager(mh, node)
+
+            org_path = f"{ou['location']}/{ou['name']}"
+
+            for uuid, employee in employees.items():
+                row = {}
+                address = mh.read_user_address(uuid, username=True, cpr=True)
+
+                """
+                Create generic mapping function from employee data to fieldnames
+                """
+
+
+def export_to_orgviewer(mh, all_nodes):
+
+    # Root must be Holstebro Kommune for org viewer
+    roots = SETTINGS['exports.holstebro.roots']
+    org_root_name = SETTINGS['municipality.name']
+    org_root_uuid = SETTINGS['municipality.uuid']
+
+    data_fields = {'name': 'Location', 'value': '', 'type': 'text'}
+    people = []
+    assignments = []
+
+    # Manually add root org unit "Holstebro Kommune" from settings and
+    # add manager to assignments and people
+    data = {'api_version': '1.0',
+            'chart': {
+                'id': org_root_uuid,
+                'name': org_root_name,
+                'description': '',
+                'parent_id': '',
+                'manager_id': SETTINGS['exports.holstebro.root_manager'],
+                'staff_department': 'N',
+                'dataFields': [data_fields],
+                'children': [],
+                "showChildren": True
+            },
+            'people': people,
+            'assignments': assignments
+            }
+
+    # this is where we start adding children
+    # Create dict with {'parent_uuid': ref to children[]}
+    children_ref = {}
+    children_ref.update({org_root_uuid: data['chart']['children']})
+
+    logger.info(f"Exporting root unit: {org_root_name} to Org. viewer")
+
+    # is this a partial exports
+    export_all = True if len(roots[org_root_uuid]) == 0 else False
+
+    logger.info(f"Exporting {org_root_name} to Org. viewer")
+
+    nodes = all_nodes[org_root_uuid]
+
+    # First create list of all children
+    # then put them in hierarchial order
+
+    for node in PreOrderIter(nodes['root']):
+        # if this is only partial export of org and this org unit is not on the exports list
+        # or is a child of an org unit in the exports list, continue
+        export_node = False
+        node_parent = node
+        while(node_parent != None and export_node is False):
```

Comment:
Man skal passe på med at bruge `is` operatoren. Den tjekke ikke logisk ækvivalens, men derimod at det underliggende objekt har samme hukommelses adresse:
```suggestion
        while(node_parent is not None and export_node == False):
```
Et eksempel på hvor det kan gå galt er:
```
>>> a = str(1)
>>> a == '1'
True
>>> a is '1'
False
```
Her har `a` samme værdi, men forskellig hukommelsesadresse, dermed svarer `is` ikke det samme som `==`.
Man kan se hukommelsesadresserne med `id`, altså: `id(a)`.
Overstående kodestump kan godt ske at svare `True` til `is`, hvis man er så heldig at Python interpreteren har optimeret det til at bruge det samme memory.

Det er altså ret uoverskueligt hvornår `is` svare sand, og hvornår den svare falsk, se f.eks. her:
```
for x in range(-10, 300):
     print(x, x + 1 is x + 1)
```
Der giver:
```
(-10, False)
(-9, False)
(-8, False)
(-7, False)
(-6, True)
(-5, True)
...
(254, True)
(255, True)
(256, False)
(257, False)
...
```
Fordi Python interpreteren behandler små integers forskelligt fra store integers (-6-->255).

TL;DR; Brug udelukkende `ìs` til at tjekke om noget er `None`.